src/Domain/FunctionsOfTime/PiecewisePolynomial.hpp

```diff
@@ -76,12 +77,17 @@ class PiecewisePolynomial : public FunctionOfTime {
   // NOLINTNEXTLINE(google-runtime-references)
   void pup(PUP::er& p) override;
 
- private:
   template <size_t LocalMaxDeriv>
   friend bool operator==(  // NOLINT(readability-redundant-declaration)
       const PiecewisePolynomial<LocalMaxDeriv>& lhs,
       const PiecewisePolynomial<LocalMaxDeriv>& rhs) noexcept;
 
+  template <size_t LocalMaxDeriv>
+  friend std::ostream& operator<<(
+      std::ostream& os,
+      const PiecewisePolynomial<LocalMaxDeriv>& piecewise_polynomial) noexcept;
+
+ private:
```

Comment:
Clang-tidy is complaining about a duplicate symbol, possibly because this is nor public rather than private? (Not fully sure)

Reply:
It's a clang-tidy bug.  Just NOLINT it like the previous function.